### PR TITLE
Add multimodel unit tests

### PR DESCRIPTION
**Pull request by valeriupredoi**
port ESMValGroup/ESMValTool#1142 into ESMValCore/development

### CODE REVIEW

**Comment by bouweandela** [comment]
Can you please merge the `development` branch into this branch again? Since #12 has been merged now, this should result in fewer changes.

**Comment by valeriupredoi** [comment]
Will do, cheers man, and will do the same for my other PR's :beer:

Dr Valeriu Predoi.
Computational scientist
NCAS-CMS
University of Reading
Department of Meteorology
Reading RG6 6BB
United Kingdom

**Comment by valeriupredoi** [comment]
le done :beer:

File: tests/unit/preprocessor/_multimodel/test_multimodel.py [hunk 1]
```diff
@@ -0,0 +1,138 @@
+"""Unit test for :func:`esmvalcore.preprocessor._volume`."""
```
Comment (bouweandela) [inline tests/unit/preprocessor/_multimodel/test_multimodel.py]:
```suggestion
"""Unit test for :func:`esmvalcore.preprocessor._multimodel`."""
```

**Comment by valeriupredoi** [comment]
@mattiarighi wanna have a testy-test before le merge? :beer:

**Comment by mattiarighi** [comment]
Yes, please.

**Pull request merged**